### ISSUE TITLE
Implement new version gulp-scss-lint

### ISSUE THREAD

**Issue by gewoonderrik** [trendwerk/trendpress#755]
As discussed: we should implement a new version of gulp-scss-lint so we can start using BEM.

**Comment by luukdv** [comment]
Maybe we should broaden the scope of this issue to cover BEM entirely? Updating classes in the theme, etc.

**Comment by gewoonderrik** [comment]
Yes. Good point! It would be nice if WordPress could output BEM classes as wel (for the menu, body class, etc).
